### PR TITLE
[vcpkg-cmake] get more logs

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

File: ports/vcpkg-cmake/vcpkg_cmake_configure.cmake [hunk 1]
```diff
@@ -100,13 +100,21 @@ function(vcpkg_cmake_configure)
...
     if(generator STREQUAL "Ninja")
         vcpkg_find_acquire_program(NINJA)
         vcpkg_list(APPEND arg_OPTIONS "-DCMAKE_MAKE_PROGRAM=${NINJA}")
         # If we use Ninja, it must be on PATH for CMake's ExternalProject,
         # cf. https://gitlab.kitware.com/cmake/cmake/-/issues/23355.
         get_filename_component(ninja_path "${NINJA}" DIRECTORY)
         vcpkg_add_to_path("${ninja_path}")
+        set(parallel_log_args
+            "../build.ninja" ALIAS "rel-ninaja.log"
+            "../../${TARGET_TRIPLET}-dbg/build.ninja" ALIAS "dbg-ninaja.log"
```
Comment (dg0yt) [inline ports/vcpkg-cmake/vcpkg_cmake_configure.cmake:115]:
```suggestion
            "../build.ninja" ALIAS "rel-ninja.log"
            "../../${TARGET_TRIPLET}-dbg/build.ninja" ALIAS "dbg-ninja.log"
```
but I'm not sure if they are so useful.

**Review by dg0yt — CHANGES_REQUESTED** [review]
`CMakeConfigureLog.yaml` is great.

Note that another change to this port is pending in #38249.

**Comment by dg0yt** [comment]
And IIUC all these logs become part of the GH issues.

File: ports/vcpkg-cmake/vcpkg_cmake_configure.cmake (hunk 1 above)
Comment (Neumann-A) [reply ports/vcpkg-cmake/vcpkg_cmake_configure.cmake]: Useful for e.g. debugging link issues where the install logs just shows the response file being used.

Comment (dg0yt) [reply ports/vcpkg-cmake/vcpkg_cmake_configure.cmake]: I understand the problem with the response files, but does the ninja file at more information? (Dependencies, of course, but content?)

**Comment by dg0yt** [comment]
meson and gn use ninja, too. At which point will "build with ninja" be generalized?

**Comment by Neumann-A** [comment]
```
REGRESSION: openmvs:x64-windows-static failed with BUILD_FAILED. If expected, add openmvs:x64-windows-static=fail to D:\a\_work\1\s\scripts\azure-pipelines/../ci.baseline.txt.
REGRESSION: rtabmap:x64-windows-static failed with BUILD_FAILED. If expected, add rtabmap:x64-windows-static=fail to D:\a\_work\1\s\scripts\azure-pipelines/../ci.baseline.txt.
```

are baseline due to the removal of CBLAS_LIBRARIES from the lapack-wrapper is fixed in the VTK PR

**Comment by Neumann-A** [comment]
I wouldn't mind `SAVE_LOG_FILES` to be exposed to the caller somehow. Maybe some `Z_VCPKG_` variable would make sense here? The main use is debugging especially for the `build.ninja` file.

**Review by FrankXie05 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the logs!

**Pull request merged**